viewer-core/src/main/java/org/edmcouncil/spec/fibo/weasel/ontology/data/OwlDataHandler.java

```diff
@@ -485,12 +492,12 @@ private IRI extractSubElementIri(OwlAxiomPropertyValue axiomProperty, IRI objIri
       Set<String> ignoredToDisplay,
       String key,
       int startCountingArgs,
-      Boolean usage
+      Boolean bypassClass
```

Comment:
Don't use `Boolean` if not need. Use simple primitive value `boolean` instead. See for example for some justifications https://stackoverflow.com/questions/3728616/boolean-vs-boolean-in-java.